iDiff/differs/package.go

```diff
@@ -75,3 +64,95 @@ func getDiffOutput(d1file, d2file string, json bool) (string, error) {
 
 	return buffer.String(), nil
 }
+
+func diffImageFiles(img1, img2 string) (utils.DirDiff, error) {
+	var diff utils.DirDiff
+	img1FS, err := utils.ImageToFS(img1)
```

Comment:
Don't worry about addressing this in this PR, but this should eventually be pulled out into an `ImagePrep` interface (or something similar) so that that logic is decoupled from the actual differs.